core/src/main/java/org/springframework/security/authentication/UserDetailsRepositoryReactiveAuthenticationManager.java

```diff
@@ -102,4 +110,16 @@ public void setUserDetailsPasswordService(
 			ReactiveUserDetailsPasswordService userDetailsPasswordService) {
 		this.userDetailsPasswordService = userDetailsPasswordService;
 	}
+
+	/**
+	 * Sets the strategy which will be used to validate the loaded <tt>UserDetails</tt>
+	 * object for the user.
+	 *
+	 * @param userDetailsChecker The {@link UserDetailsChecker}
+	 * @since 5.2
+	 */
+	public void setUserDetailsChecker(UserDetailsChecker userDetailsChecker) {
```

Comment:
We should expose this as `setPostAuthenticationChecks` to align with `AbstractUserDetailsAuthenticationProvider`. We should also update the member variable to `postAuthenticationChecks`